modules/core/src/main/scala/com/azavea/stac4s/core/StacCatalog.scala

```diff
@@ -12,16 +13,17 @@ final case class StacCatalog(
 
 object StacCatalog {
 
+  implicit val eqStacCatalog: Eq[StacCatalog] = Eq.fromUniversalEquals
+
   implicit val encCatalog: Encoder[StacCatalog] =
-    Encoder.forProduct5("stac_version", "id", "title", "description", "links")(
-      catalog =>
-        (
-          catalog.stacVersion,
-          catalog.id,
-          catalog.title,
-          catalog.description,
-          catalog.links
-        )
+    Encoder.forProduct5("stac_version", "id", "title", "description", "links")(catalog =>
+      (
+        catalog.stacVersion,
+        catalog.id,
```

Comment:
Looks like spec has `stac_extensions` on catalog level too, though, similar to the collection and item levels, this field is not a required one.